SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1419,16 +1490,17 @@ void USpatialReceiver::QueueIncomingRepUpdates(FChannelObjectPair ChannelObjectP
 	}
 }
 
-void USpatialReceiver::QueueIncomingRPC(const TSet<FUnrealObjectRef>& UnresolvedRefs, UObject* TargetObject, UFunction* Function, RPCPayload& Payload, const FString& SenderWorkerId)
+void USpatialReceiver::QueueIncomingRPC(FPendingRPCParamsPtr Params)
 {
-	TSharedPtr<FPendingIncomingRPC> IncomingRPC = MakeShared<FPendingIncomingRPC>(UnresolvedRefs, TargetObject, Function, Payload);
-	IncomingRPC->SenderWorkerId = SenderWorkerId;
-
-	for (const FUnrealObjectRef& UnresolvedRef : UnresolvedRefs)
+	if (!Params->TargetObject.IsValid())
 	{
-		FIncomingRPCArray& IncomingRPCArray = IncomingRPCMap.FindOrAdd(UnresolvedRef);
-		IncomingRPCArray.Add(IncomingRPC);
+		// Target object was destroyed before the RPC could be (re)sent
```

Comment:
destroyed or removed?

Reply:
Destroyed, since we check the WeakPtr to it.

Reply:
Resolving since there's no pending action.